AppCenter/AppCenterTests/MSTicketCacheTests.m

```diff
@@ -0,0 +1,69 @@
+#import <XCTest/XCTest.h>
+
+#import "MSTicketCache.h"
+
+@interface MSTicketCacheTests : XCTestCase
+
+@property(nonatomic) MSTicketCache *sut;
```

Comment:
what is "sut"?

Reply:
@jwallra https://en.wikipedia.org/wiki/System_under_test

Reply:
I prefer for it to be written out then ;)

Reply:
Writing `sut` is a throughout the SDK as a convention. I'd rather keep `sut`.